airgun/entities/host.py

```diff
@@ -84,6 +84,13 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def read_filled_searchbox(self):
+        """Read filled searchbox"""
+        view = self.navigate_to(self, 'All')
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        view.wait_displayed()
+        return view.searchbox.read()
+
```

Comment:
These changes are working absolutely fine, I tested it locally, and now running in CI as part of this robottelo PR https://github.com/SatelliteQE/robottelo/pull/16641
These changes are here in another airgun PR https://github.com/SatelliteQE/airgun/pull/1593